src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/SessionContextValueResolver.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Controller\ArgumentResolver;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Attribute\MapSessionContext;
+use Symfony\Component\HttpKernel\Controller\ValueResolverInterface;
+use Symfony\Component\HttpKernel\ControllerMetadata\ArgumentMetadata;
+
+final class SessionContextValueResolver implements ValueResolverInterface
+{
+    public function resolve(Request $request, ArgumentMetadata $argument): array
+    {
+        if (!$request->hasSession()) {
+            return [];
+        }
+
+        $attribute = $argument->getAttributesOfType(MapSessionContext::class, ArgumentMetadata::IS_INSTANCEOF)[0]
+            ?? null;
+
+        if (!$attribute) {
```

Comment:
```suggestion
        if (!$attribute = $attribute = $argument->getAttributesOfType(MapSessionContext::class, ArgumentMetadata::IS_INSTANCEOF)[0] ?? null) {
```

Reply:
There's a typo: `$attribute = $attribute = ...`
It should be like this:

```suggestion
        if (!$attribute = $argument->getAttributesOfType(MapSessionContext::class, ArgumentMetadata::IS_INSTANCEOF)[0] ?? null) {
```